saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleHandler.java

```diff
@@ -207,13 +211,13 @@ private void updateAbrp(
               }
             };
         String execute = httpclient.execute(httppost, responseHandler);
-        System.out.println("ABRP: " + execute);
+        LOGGER.info("ABRP: " + execute);
```

Comment:
I think this should be debug. It usually only contains "OK". And it's output is quite often.